d/dktest/dktest_ubi_8.3.sh

```diff
@@ -0,0 +1,72 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : dktest
+# Version               : 0.3.0,0.3.6
+# Source repo           : https://github.com/dhui/dktest
+# Tested on             : UBI 8.3
```

Comment:
Logs show this was tested on ubuntu. Pl. clarify whether it is tested on ubi, ubuntu or both ?

Reply:
If final script tested on ubuntu, pl. update the header comment accordingly

Reply:
done ,thanks.Pls review now